Exiled.Events/EventArgs/ElevatorTeleportEventArgs.cs

```diff
@@ -42,9 +42,19 @@ public ElevatorTeleportEventArgs(Player player, Camera079 camera, float apCost,
         public Camera079 Camera { get; set; }
 
         /// <summary>
-        /// Gets or sets the amount of AP will be consumed during the level change.
+        /// Gets or sets the amount of auxiliary power required to interact with a tesla gate.
         /// </summary>
-        public float APCost { get; set; }
+        public float AuxiliaryPowerCost { get; set; }
+
+        /// <summary>
+        /// Gets or sets the amount of AP that will be removed for the first time when using speakers through SCP-079.
```

Comment:
Ugh...

Reply:
Do I have to change that description? It's an obsolete property after all.

Reply:
I can say yeah, even if a thing is deprecated, we should maintain full documentation for it.